signbank/dictionary/csv_import.py

```diff
@@ -259,274 +271,277 @@ def confirm_import_nzsl_share_gloss_csv(request):
         # Set a message to be shown so that the user knows what is going on.
         messages.add_message(request, messages.WARNING, _("Cancelled adding CSV data."))
         return HttpResponseRedirect(reverse("dictionary:import_nzsl_share_gloss_csv"))
+    elif not "confirm" in request.POST:
```

Comment:
**praise**: thank you for moving these around, I'm feeling like it's karma that I suggested it and then needed to meticulously scan through the diff to check for changes 😅

Reply:
I think I had originally changes it in the PR when I first added the view, but then messed it back up with some rebasing on other branches 😅